app/src/test/java/com/eis/geoCalendar/app/network/GenericEventNetworkTest.java

```diff
@@ -0,0 +1,52 @@
+package com.eis.geoCalendar.app.network;
+
+import android.location.Location;
+
+import com.eis.communication.Peer;
+import com.eis.geoCalendar.gps.GPSPosition;
+import com.eis.geoCalendar.network.EventNetworkManager;
+import com.eis.geoCalendar.network.NetworkEvent;
+import com.eis.geoCalendar.network.NetworkEventUser;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * @author Luca Crema
+ * @since 26/12/2019
+ */
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class should define a constructor.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4741777)

Reply:
Nah it couldn't, it's a test